doc/enterprise-shield/c_enterprise_shield_use_cases.md

```diff
@@ -0,0 +1,78 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Common Enterprise Shield™ Use Cases ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=====================================
+
+This topic provides a high-level overview of some common Enterprise Shield™ use cases.
+
+Before You Begin
+--------------------------------------
+
+This procedure is part of [Configure Enterprise Shield™ for KAAZING Gateway](o_enterprise_shield_checklist.md):
+
+1.  [Common KAAZING Gateway Production Topologies](../admin-reference/c_topologies.md#enterprise-shield--)
+2.  [About Enterprise Shield™](o_enterprise_shield_checklist.md#about-enterprise-shield)
+3.  **Common Enterprise Shield™ Use Cases**
+4.  [Setting Up the Gateway](../about/setup-guide.md)
+5.  [Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md)
+6.  [Walkthrough: Configure Enterprise Shield™ for High Availability](p_enterprise_shield_cluster.md)
+
+The use case configurations described in this topic vary depending on the level of security, such as if the Gateways are clustered for high availability, whether network transports are used, and so on. Also, the sample topologies use two Gateway services instead of just one to demonstrate the Enterprise Shield™ reverse connectivity behavior in a configuration with multiple services.
+
+Each of the Enterprise Shield™ use cases configures Enterprise Shield™ on a single pair of Gateways (DMZ and internal Gateways), and then extends that topology to a cluster configuration. To configure Enterprise Shield™ and control Gateway operation, you configure elements in the Gateway configuration (gateway-config.xml) file. For complete information about Gateway configuration, see [About Gateway Configuration](../admin-reference/c_configure_gateway_concepts.md). Also, the complete Gateway configuration files used for these use cases are available on `kaazing.org`. The configuration file names are included in each use case description that follows.
```

Comment:
This is being tracked in [#252 Package Enterprise Edition Documentation](https://github.com/kaazing/roadmap/issues/252). 

" Ensure that kaazing.org directs EE users to EE samples, such as the Enterprise Shield and Clustering configs. Other samples are available in CE/open in GitHub."